ivy/functional/frontends/torch/reduction_ops.py

```diff
@@ -53,7 +53,9 @@ def sum(input, dim=None, keepdim=False, *, dtype=None, out=None):
 
 
 @to_ivy_arrays_and_back
-def mean(input, dim=None, keepdim=False, *, out=None):
+def mean(input, dim=None, axis=None, keepdim=False, *, out=None):
+    if dim is None:
+        dim = axis
     return ivy.mean(input, axis=dim, keepdims=keepdim, out=out)

```

Comment:
What happens if both `dim` and `axis` are given? Is there an assertion in the original torch function?

Reply:
@AnnaTz I *think* that torch does this with function overloading in C++. If you try to call `torch.mean(x, dim=-1, axis=-1)` then that doesn't work in native torch. So I think this change would be fine.